Update some docs to reflect changes that have happened lately. (#56)
NordicSemiconductor/cddl-gen: README.md

Running tests
=============

> The tests for the generated code is based on the same way as Zephyr ztests.

Reviewer: ```suggestion
The tests for the generated code are based on the same method as Zephyr ztests.
```

Author: ```suggestion
The tests for the generated code are based on Zephyr ztests.
```
